frontend/src/components/events.jsx

```diff
@@ -1,78 +1,111 @@
-                <p>Jun&nbsp; 2018 – Almost Grown-ish&nbsp;</p>
-                <p>May 2018 – Urban Listening Retreat&nbsp;</p>
-                <p>May 2018 – Prospancia Leadership: “Disruptive Women”</p>
-                <p>May 2018 – Mothers’ Day Brunch</p>
-                <p>May 2018 – MillerCoors Business and Beer</p>
-                <p>May 2018 – Columbia College – OWN It!</p>
-                <p>May 2018 – Community Organizing&nbsp;</p>
-                <p>May 2018 – Google Presentation</p>
-                <p>May 2018 – Barbara Bates Design &amp; Entrepreneurial Workshop</p>
-                <p>Apr. 2018 – Women Thriving Fearlessly</p>
-                <p>Apr. 2018 – International trip to Dubai</p>
-                <p>Apr. 2018 – NEIU Spiritual/Mental Health Panel</p>
-                <p>Mar. 2018 – Spring Leadership Meeting – Part 2</p>
-                <p>Feb. 2018 – Evanston, IL – Levy Center, Le Tour De Noir Black Business Expo – Associate Board and Volunteer Recruitment</p>
-                <p>Feb. 2018 – Evanston, IL – LePeep Restaurant – Scholarship Council Meeting</p>
-                <p>Feb. 2018 – Oak Park, IL – Chicken &amp; Waffles Restaurant – Marketing Round table</p>
-                <p>Jan. 2018 – Chicago, IL – Rogers Park Library – Spring Leadership Meeting part 1 | Logic Model Presentation with guest facilitator: Anne Gramiak (sold out)</p>
-                <p><strong>2017 Past Events</strong></p>
-                <p>May 2017 – Chicago, IL – Pinstripes Restaurant – Chicago Community Trust – On The Table Events (2-day events)</p>
-                <p>Oct. 2017 – Chicago, IL – University of Illinois at Chicago – Community Focus Groups (2-day events)</p>
-                <p>Oct. 2017 – East Chicago, IN – East Chicago Public Library on Columbus Ave. – Leadership Meeting (1-day event)</p>
-                <p>In efforts to prepare for our upcoming cohort, we host community focus group events with alumnae and public individuals from the Chicagoland, Northwest Indiana, and Wisconsin region.</p>
-                <p>Young adult men and women who have experienced life as either an orphan, adoptee, or foster youth alumni share their diverse voices and stories on what they feel is needed or lacking for youth in care, and what they feel is needed now to help young women stand strong and thrive. We will continue listening to alumni and strive to create programs that matter.</p>
-                <p><a href="/contact-us/">Contact us now!</a></p>
-                <p>Our 2019-2020 co-hort participant recruitment campaign is now underway.&nbsp;&nbsp;<strong>The YMIM</strong> is seeking&nbsp;<a href="/ambassadors/">volunteer ambassadors</a> to support the cohort.&nbsp;Join our movement to inspire, connect, and empower young adult women orphans, adoptees, and foster youth alumnae. Now and always, your generous contributions are greatly appreciated!</p>
+        return (
```

Comment:
Rather than just having this list here, can you convert this to a fixture to load, remove the existing events from the page, and display these from the DB?

Reply:
done